libs/platform/CMakeLists.txt

```diff
@@ -106,6 +106,7 @@ target_link_libraries("platform_kernel" PRIVATE
 )
 
 target_compile_definitions("platform_kernel" PRIVATE
+  KERNEL_MODE
```

Comment:
Do we need `KERNEL_MODE` also when user mode is defining `USER_MODE`?

Reply:
I think it's more clear, and usable for the future.